docs/website/views/_includes/analytics.njk

```diff
@@ -0,0 +1,9 @@
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-FTFSXX8LXV"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
```

Comment:
could you add formatting, please?

Reply:
That's the snippet from Google Tags, but ok